### DISCUSSION TITLE
Disabling select menu in V2.0+

### DISCUSSION THREAD

**Discussion by codeofandrin** [Rapptz/discord.py#7201] [Ideas]
Will there be the opportunity in the near future to disable a select menu like at the button? Means that when a user choose one element of the select menu that he's not be able to select another one anymore.

**Comment by Rapptz** [comment]
Discord added disabling selects from feedback I gave them during the alpha/beta testing period for selects. I just haven't gotten around to implementing it. It's also undocumented.

**Reply by jelni** [reply]
Ok, sorry for that. I will remove my comment

**Reply by codeofandrin** [reply]
Thanks for your answer!
